lib/Transforms/Instrumentation/ExtendedPointer.cc

```diff
@@ -264,13 +264,13 @@ bool ExtendedPointer::runOnFunction(Function &F) {
 
```

Comment:
the change to this file should be split from this commit and merge into the previous one